src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -844,14 +843,14 @@ it can also be launched from different Proactive portals.
 ==== Via Proactive Studio Portal
 The data drift detection function is added to the tasks and workflows of the bucket
 *model_as_a_service*. This function is directly linked to the deployment of the model
-in MaaS_ML (where the user deploys the model, its metadata and a part of the
-old training data set to be used for drift detection in the new input data set),
+in MaaS_ML (where the user deploys the model and a part of the
+old training dataset to be used for drift detection in the new input data set),
 and to the call of the prediction service in MaaS_ML (where the drift detector is chosen
-and the detection process is implemented using the chosen detector).
+and the detection process is launched using the chosen detector).
 
 In the workflow named *IRIS_Deploy_Predict_Flower_Classifier_Model* found in the
 *model_as_a_service* bucket in the Proactive Studio Portal, the task
-*Deploy_ML_Model* and *Call_Prediction_Endpoint* have been mainly modified to include the DDD
+*Deploy_ML_Model* and *Call_Prediction_Service* have been mainly modified to include the DDD
 features.
```

Comment:
The workflow *IRIS_Deploy_Predict_Flower_Classifier_Model*, found in the
*model_as_a_service* bucket in the Proactive Studio Portal, shows an example of pipeline using the generic tasks 
*Deploy_ML_Model* and *Call_Prediction_Service* including the DDD mechanism.